task-3_mongoDB-p1/README.md

```diff
@@ -0,0 +1,92 @@
+```
+mongoimport --db frontcamp --collection restaurants --file C:/data/db/restaurants.json
+```
+
+# task 3:
+
+1) RESULT: 728
+---------------
+
+```
+{ borough: "Queens", cuisine: "Chinese"}
+```
+
+2) RESULT: 5dc468ae4c745d010cdeb463 "restaurant_id": "40372466"
+---------------
+
+```
+$project {
+  _id: "$_id",
+  maxScore: {$max: "$grades.score"}
+} + $sort {
+  maxScore: -1
+}
+```
+
+3) Done
+---------------
+```
+{ borough: "Manhattan"}, { $set: { "grades": { grade : "A", score: 7, date: ISODate() }}}
+```
+
+4) RESULT: name:"Silver Krust West Indian Restaurant"
+---------------
+
+```
+filter:{ "grades.8.score": {$lt: 7}} + project: {name: 1}
+```
+
+5) RESULT: { _id: "5dc468af4c745d010cdee854" ,borough: "Bronx"}
+---------------
```

Comment:
I guess these are not all ids

Reply:
I checked one more time but I have found only one item for criterias: { cuisine: "Seafood", ".grade": "B", period from "2014-02-01T00:00:00.000Z" to"2014-03-01T00:00:00.000Z"}

Reply:
I recommend redo the query, the answer isn't entire

Reply:
I used query such as { cuisine: "Seafood", "grades.grade": "B", "grades.date": {$gte: ISODate("2014-02-01T00:00:00.000Z"), $lt: ISODate("2014-03-01T00:00:00.000Z")}}

Reply:
Use something like as `>db.restaurants.find( "cuisine": "Seafood", "grades": {$elemMatch: {...}})`

Reply:
![image](https://user-images.githubusercontent.com/45118964/68549108-a2267b80-0405-11ea-92a1-23360ca6b6b8.png)